sdk/java/src/main/java/com/pulumi/docker/Image.java

```diff
@@ -42,7 +43,10 @@
  * 
  *     public static void stack(Context ctx) {
  *         var demoImage = new Image(&#34;demoImage&#34;, ImageArgs.builder()        
- *             .build(%!v(PANIC=Format method: interface conversion: model.Expression is *model.TemplateExpression, not *model.LiteralValueExpression))
+ *             .build(DockerBuildArgs.builder()
```

Comment:
well this is an improvement! 🙌 
there is still something wonky with string conversion here but that should be a followup